MAUI/Calendar/migration.md

```diff
@@ -1105,7 +1105,7 @@ this.Content = calendar;
 <tr>
 <td>{{'[NavigateTo](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_NavigateTo_System_DateTime_)'| markdownify }}</td>
 <td>Nil</td>
-<td>Not Supported.</td>
+<td>Using [DisplayDate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_DisplayDate), you can navigate to specific date.</td>
 </tr>
 <tr>
```

Comment:
AddDatesInPast | Nil | Not Supported.
-- | -- | --


[AddDatesInPast](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_AddDatesInPast)	Nil	Not Supported.

Can suggest enable dates in past?